### ISSUE TITLE
Shoul be a possiblity to adjust win/canvas resolution on startup

### ISSUE THREAD

**Issue by trsh** [Kode/Kha#134]
Disar: resizing the framebuffer dynamically is another option that would be great. But all in due time i guess 

Robert: game window resize is a technical problem, because D3D makes that hard. But will have to solve that eventually. 

trsh: Robert: but the current code doe's load the size from Loader. We just need a place to inject our recalculated values 
In a nice / not hack way 

Disar: trsh: i think it would be easier if there is an event you could subcribe to and when it's called you reread the width and height trough sys 
unless that doesn't work. 
then i don't know. 

Robert: Delaying window creation so it can be triggered by code wouldn't be hard to implement, but I'll probably tackle all of it at once so that the api changes only once. 

wighawag: that would be good, I am not using native yet but when time will come resizable windows would be nice to have

**Comment by RobDangerous** [comment]
Hey, this is fixed.

**Comment by trsh** [comment]
Nice!
